scripts/ftc_forms.js

```diff
@@ -238,7 +238,18 @@ FTC.forms.edit_item_fields = function(html, character) {
     });
 };
 
-
+/* -------------------------------------------- */
+/* Buttons                                      */
+/* -------------------------------------------- */
+FTC.forms.use_button_fields = function(html, entity) {
```

Comment:
This is ok, and to a certain extent you were following my own (poor) design - but one improvement to maybe think about, if these handlers are specific to the Actor sheet (which I think they are) - it might make more sense to just define them within the `FTCCharacter.activateListeners()` method rather than applying this to all forms.